config/project.ini

```diff
@@ -209,10 +209,10 @@ ckan.comments.follow_mute_enabled = True
 ckan.comments.show_comments_tab_page = True
 
 ##Use the following configuration options to choose the operation modes:
-ckanext.validation.run_on_create_async = True
-ckanext.validation.run_on_update_async = True
-ckanext.validation.run_on_create_sync = False
-ckanext.validation.run_on_update_sync = False
+ckanext.validation.run_on_create_async = False
+ckanext.validation.run_on_update_async = False
+ckanext.validation.run_on_create_sync = True
+ckanext.validation.run_on_update_sync = True
```

Comment:
We can't have this enabled by default. Please make it swap programmatically where required. (i.e. when a validation json file is attached to the resource and/or pii is configured to hide when invalid)

We require a response within 60seconds as this site is behind cloudfront CDN. 

Example dataset/resource that will fail is https://www.data.qld.gov.au/dataset/queensland-covid-19-case-line-list-location-source-of-infection/resource/1dbae506-d73c-4c19-b727-e8654b8be95a which has about 704000 records.

Reply:
Will be fixed and pushed by Alex shortly.